src/gam.py

```diff
@@ -7000,11 +7000,13 @@ def doCreateGroup():
       i += 2
     elif myarg == u'description':
       description = sys.argv[i+1].replace(u'\\n', u'\n')
-      if description.find(u'\n') != -1:
-        gs_body[u'description'] = description
-        if not gs:
-          gs = buildGAPIObject(u'groupssettings')
-          gs_object = gs._rootDesc
+      for c in u'\n<>=':
```

Comment:
looks fine but can you add a brief comment just above as to why this is necessary so it will jog our memory in the future?